tox.ini

```diff
@@ -23,6 +23,9 @@ deps =
     oldestdeps: matplotlib==3.5.0
     figure-!devdeps-!oldestdeps: astropy==5.3.4
     figure-!devdeps-!oldestdeps: matplotlib==3.8.1
+    # Due to https://github.com/matplotlib/pytest-mpl/issues/216
+    # Needs a release of pytest-mpl with the fix
+    figure: pluggy<1.4
```

Comment:
but we don't have an issue with pytest-mpl?

Reply:
True, I don't know why.

Reply:
I removed it.